packages/next-app/src/app/stableswapsimulator/(components)/TokenTable.tsx

```diff
@@ -40,6 +40,8 @@ export function TokenTable({
     const { variantData } = useStableSwap();
     tableData = variantData;
   }
+
+  const aboveOrEqualLimit = tableData?.tokens?.length >= 8;
```

Comment:
you'd help your reader by setting `8` to a constant defined outside of this component, with a comment briefly explaining where it's coming from (I know it's an implementation limit, but not sure which contract that's defined in)